integrations-and-supported-tools/pytorch-ignite/notebooks/Neptune_PyTorch_Ignite.ipynb

```diff
@@ -0,0 +1,704 @@
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "lL3OBjbqeluh"
+   },
+   "outputs": [],
+   "source": [
+    "optimizer = SGD(model.parameters(), lr=params[\"lr\"], momentum=params[\"momentum\"])\n",
+    "criterion = nn.CrossEntropyLoss()\n",
+    "\n",
+    "trainer = create_supervised_trainer(model, optimizer, criterion, device=device)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "FtveI27Yelui",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## (Neptune) Create NeptuneLogger()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "_9X4eCjGelui"
+   },
+   "outputs": [],
+   "source": [
+    "import neptune\n",
+    "from ignite.contrib.handlers.neptune_logger import NeptuneLogger\n",
+    "\n",
+    "neptune_logger = NeptuneLogger(\n",
+    "    api_token=neptune.ANONYMOUS_API_TOKEN,\n",
+    "    project=\"common/pytorch-ignite-integration\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "9x-KVoHJeluj"
+   },
+   "source": [
+    "To open the run, click the Neptune link that appears in the console output. This will be updated live once training starts."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "hu1pkNFQeluj"
+   },
+   "source": [
+    "## (Neptune) Attach logger to the trainer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "0Dy0htR0eluk"
+   },
+   "outputs": [],
+   "source": [
+    "trainer.logger = setup_logger(\"Trainer\")\n",
+    "\n",
+    "neptune_logger.attach_output_handler(\n",
+    "    trainer,\n",
+    "    event_name=Events.ITERATION_COMPLETED(every=100),\n",
+    "    tag=\"training\",\n",
+    "    output_transform=lambda loss: {\"batchloss\": loss},\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "o4hrvDrheluk"
+   },
+   "source": [
+    "## Create evaluators"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "P5SPE_e-eluk"
+   },
+   "outputs": [],
+   "source": [
+    "metrics = {\"accuracy\": Accuracy(), \"loss\": Loss(criterion)}\n",
+    "\n",
+    "train_evaluator = create_supervised_evaluator(model, metrics=metrics, device=device)\n",
+    "\n",
+    "validation_evaluator = create_supervised_evaluator(model, metrics=metrics, device=device)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "jzw-MQcrelul"
+   },
+   "outputs": [],
+   "source": [
+    "@trainer.on(Events.EPOCH_COMPLETED)\n",
+    "def compute_metrics(engine):\n",
+    "    train_evaluator.run(train_loader)\n",
+    "    validation_evaluator.run(val_loader)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "8wiMykjielul"
+   },
+   "source": [
+    "### (Neptune) Attach logger to training and validation evaluators"
```

Comment:
Everyother heading looks to use `##` except this one? Intentional? It looks a bit odd while looking at the notebook.